content/citation-tags.tsv

```diff
@@ -47,6 +47,7 @@ CudNN	arxiv:1410.0759
 Dahl2014_multi_qsar	arxiv:1406.1231
 Dean2012_nips_downpour	url:http://research.google.com/archive/large_deep_networks_nips2012.html
 DeepChem	url:https://github.com/deepchem/deepchem
+DeepSpeech	arxiv:1412.5567
```

Comment:
This tag isn't used.  Can we remove it?  I think the paper was used in an edit to the introduction in #636, but it didn't look critical to me.

Reply:
Sure